go.mod

```diff
@@ -1,6 +1,6 @@
 module gitsnap
 
-go 1.16
+go 1.14
```

Comment:
This was a mistake I made before, we don't need to update the version of go itself... reverting

Reply:
though we probably should upgrade go... probably version 1.14 has vulns, its pretty old. but we can do it later

Reply:
this is binary... it's unrelated to the runtime :)